contracts/Oracle.sol

```diff
@@ -49,35 +49,6 @@ contract Oracle is DSMath {
     function bill() public view returns (uint256) {
         return pmt;
     }
-    
-    function pack(ERC20 tok_) {
-        pack(uint128(bill()), tok_);   
-    }
-    
-    function pack(uint128 pmt_, ERC20 tok_) { // payment
-        require(uint32(now) > lag);
-        pmt = pmt_;
-        dis = 0;
-        lag = uint32(now) + DELAY;
-        owed = msg.sender;
-        tok = tok_;
-        told = false;
-        posted = false;
-        call();
-        chec();
-    }
-    
-    function call()
```

Comment:
Remove unnecessary `call` that is overridden by `Oraclize.sol` and `Chainlink.sol`